site/en/docs/extensions/mv3/manifest/file_handlers/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: "layouts/doc-post.njk"
+title: "file_handlers"
+seoTitle: "Chrome Extensions Manifest: file_handlers"
+date: 2023-11-01
+description: Reference documentation for the externally_connectable property of manifest.json.
+---
+
```

Comment:
Can we add an aside about availability here? I think identical wording to what I used here would work well: https://developer.chrome.com/docs/extensions/reference/readingList/

Reply:
I added it below the opening paragraph.

We need a way to automate stuff like this. Such comments have a way of sticking around way past their expiration dates.